### ISSUE TITLE
New Snort build - installation failure

### ISSUE THREAD

**Issue by charlesfair** [CyberAnalyticDevTeam/SimpleRock#25]
Installation of SimpleRock failed with non-vagrant VM method.  Used:
- Fusion Pro
- CentOS 7.2 min.
- 4 cores
- 8GB RAM

Beginning of installation failure as follows:  

+++++++++++++++++++++++++++++++++++++++++++
* execute[start_bro] action run
=================================================
    Error executing action `run` on resource 'execute[start_bro]'
================================================
    Mixlib::ShellOut::ShellCommandFailed
+++++++++++++++++++++++++++++++++++++++

Attached is the full transcript:  

[SimpleRock_Snort-Build_install_failure.txt](https://github.com/CyberAnalyticDevTeam/SimpleRock/files/101619/SimpleRock_Snort-Build_install_failure.txt)

**Comment by charlesfair** [comment]
As per Jeff's recommendations, I re-installed a fresh vm, and tried the Chef install again, and it worked. I used the same settings as above.   Interestingly enough, upon first install, I had some errors for Bro.  I tried rock_stop and rock_start and didn't fix the problem.  I manually ran broctl, then check+deploy+check+top+exit.  Ran rock_start again and everything looked to be working fine.  Ran Kibana and after running tcpreplay with some pcap, everything seemed to work.
